src/status/chat.nim

```diff
@@ -78,9 +83,23 @@ proc join*(self: ChatModel, chatId: string, chatType: ChatType) =
 
   self.events.emit("channelJoined", ChannelArgs(chat: chat))
 
+proc getInstalledStickers*(self: ChatModel): seq[StickerPack] =
```

Comment:
here, it would be better to pass the address as parameter instead of relying on an "instance" variable,

```nim
proc getInstalledStickers*(self: ChatModel, address: string): seq[StickerPack] =
```

then the `currentAccount` / `WalletAccount` disappears from this file